sqlglot/expressions.py

```diff
@@ -3960,7 +3966,7 @@ def else_(self, condition: ExpOrStr, copy: bool = True, **opts) -> Case:
 
 
 class Cast(Func):
-    arg_types = {"this": True, "to": True, "format": False}
+    arg_types = {"this": True, "to": False, "format": False}
```

Comment:
why did you change this, this is probably not correct

Reply:
I had a CONVERT test case out of place and fell into changing this. I'll revert.